src/qibo/backends/__init__.py

```diff
@@ -147,24 +181,32 @@ def create(self, dtype):
 
 
 def get_backend():
-    return str(GlobalBackend())
+    return str(_Global.get_backend())
 
 
 def set_backend(backend, **kwargs):
-    GlobalBackend.set_backend(backend, **kwargs)
+    _Global.set_backend(backend, **kwargs)
+
+
+def get_transpiler():
+    return str(_Global.get_transpiler())
+
+
+def set_transpiler(transpiler):
+    _Global.set_transpiler(transpiler)
 
 
 def get_precision():
-    return GlobalBackend().precision
+    return _Global.get_backend().precision
 
 
 def set_precision(precision):
-    GlobalBackend().set_precision(precision)
-    matrices.create(GlobalBackend().dtype)
+    _Global.get_backend().set_precision(precision)
+    matrices.create(_Global.get_backend().dtype)
 
 
 def get_device():
-    return GlobalBackend().device
+    return _Global.get_backend().device
```

Comment:
If you know what they are about, please provide docstrings (sorry to be annoying :bow:)

Reply:
Added docstrings.